lib/matplotlib/quiver.py

```diff
@@ -85,6 +85,13 @@
     height of the axes, respectively, when the the window is resized;
     for 'dots' or 'inches', resizing does not change the arrows.
 
+  *orientation*: ['absolute' | 'relative']
```

Comment:
Would it make more sense to call this kwarg `reference_frame` or `coord_system` and let it be` 'axes'` or `'figure'`  ?  That makes it clear that this as about controlling  how the angles are computed and matches (sort of) the patterns we use for specifying how to interpret the units on text/annotation locations.